errdefs/errors.go

```diff
@@ -27,6 +27,9 @@ var (
 
 	// ErrUnsupported is returned when a compose project uses an unsupported attribute
 	ErrUnsupported = errors.New("unsupported attribute")
+
+	// ErrIncompatible is returned when a compose project uses an incompatible attribute
+	ErrIncompatible = errors.New("incompatible attribute")
```

Comment:
How does this differ to unsupported? Is it that there are a combination of features being used that does not work?